bot/docker/bootstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Add build deps
+apk --update add --virtual build-dependencies gcc musl-dev libffi-dev openssl-dev
+
+# Install python dependencies
+cd /src/tools && python setup.py install
+cd /src/bot && python setup.py install
+
+# Cleanup
+apk del build-dependencies
```

Comment:
Is this going to remove all of `gcc musl-dev libffi-dev openssl-dev`?

Reply:
I'm assuming yes, but I'm not so familiar with `apk`.

Reply:
[Yeap](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#virtual-packages) !